### PR TITLE
update orientation.md and treehouses-remote.md (fixes #66)

### LINKED ISSUE #66: step9: rate treehouses remote on play store before changing to beta mode
I suggest that we add this either in the body of Step 9, or at the top of orientation.md

### PR DESCRIPTION
**Pull request by CalebProvost**
https://raw.githack.com/treehouses/treehouses.github.io/rateapp/#!pages/vi/orientation.md

### CODE REVIEW

**Review by XavierElon — APPROVED** [review]
Nice addition

File: pages/vi/treehouses-remote.md [hunk 1]
```diff
@@ -10,6 +10,9 @@ On an Android device, find the [treehouses remote](https://play.google.com/store
 
 ![](images/remoteiconsml.jpg)
 
+_**Rate the app**_.
+Follow [this link](https://play.google.com/apps/testing/io.treehouses.remote) to join and download the beta version.
```
Comment (dogi) [inline pages/vi/treehouses-remote.md]: hmm this is a not beta right ;)

we switch to beta in the orientation

**Review by dogi — APPROVED** [review]

**Comment by dogi** [comment]
@CalebProvost maybe you change my comment

File: pages/vi/treehouses-remote.md (hunk 1 above)
Comment (CalebProvost) [reply pages/vi/treehouses-remote.md]: ?
it leads me to beta:
image

**Comment by CalebProvost** [comment]
> @CalebProvost maybe you change my comment

So... I'm confused. Is this good to merge?

**Comment by dogi** [comment]
> > @CalebProvost maybe you change my comment
> 
> So... I'm confused. Is this good to merge?

yes ... it is better then before
except in treehouses-remote.md if they follow the instructions they will install our official google play version ... in the general scheme a still a beta ... but compared to our official google play beta version which they switch to in the orientation.md this could get confusing

File: pages/vi/treehouses-remote.md (hunk 1 above)
Comment (dogi) [reply pages/vi/treehouses-remote.md]: ya you flipped the switch already

**Comment by CalebProvost** [comment]
> > > @CalebProvost maybe you change my comment
> > 
> > 
> > So... I'm confused. Is this good to merge?
> 
> yes ... it is better then before
> except in treehouses-remote.md if they follow the instructions they will install our official google play version ... in the general scheme a still a beta ... but compared to our official google play beta version which they switch to in the orientation.md this could get confusing

Okay. So remove the instructions to install the beta from treehouses-remote.md?

File: pages/vi/treehouses-remote.md (hunk 1 above)
Comment (dogi) [reply pages/vi/treehouses-remote.md]: ooh ... I see now ... they should install official version first and then switch in a later (orientation) step to testing

Comment (CalebProvost) [reply pages/vi/treehouses-remote.md]: reverted back

**Pull request merged**